lib/utilities/reportHandler.js

```diff
@@ -34,13 +34,15 @@ function hasWSOptionalDependencies() {
 }
 
 function getCapabilities() {
+    const localVolumeCap = process.env.LOCAL_VOLUME_CAPABILITY;
     return {
         locationTypeDigitalOcean: true,
         locationTypeS3Custom: true,
         locationTypeSproxyd: true,
         locationTypeNFS: true,
         locationTypeCephRadosGW: true,
         locationTypeHyperdriveV2: true,
+        locationTypeLocal: localVolumeCap === undefined ? true : JSON.parse(localVolumeCap),
```

Comment:
No more need to check undefined just compare the string, I always add a toLower/toUpper to prevent some typos
```suggestion
        locationTypeLocal: localVolumeCap.toLowerCase() === 'true',
```